zellij-server/src/lib.rs

```diff
@@ -114,6 +116,7 @@ pub(crate) enum SessionState {
 }
 
 pub fn start_server(os_input: Box<dyn ServerOsApi>, socket_path: PathBuf) {
+    info!("starts server");
```

Comment:
No idea why this (and `Zellij logger initialized`) are printed twice :|

Reply:
Hmm, I may ping @kunalmohan since he did a lot of the daemonization stuff. This could be because that process requires forking twice, but that's just a shot in the dark!

Reply:
@qepasa did you mean `Starting Zellij!` (instead of `starts server`)?

I can see why this happens. When the user starts zellij, the control enters the `main()` function which calls `start_client()`. `start_client()` then optionally spawns a new server (if a new sessions is to be started). The way this is done is- we spawn a new zellij process with `zellij --server <socket-path>` ([here](https://github.com/zellij-org/zellij/blob/3df0210647e65170576f1ea97328cce26c211a43/zellij-client/src/lib.rs#L62-L65)). So the main function is called again, but this time it further calls `start_server()`.

We can see that the control goes to the `main()` function twice (in two separate processes). Since both the above logs are printed in main, we see them twice.

Reply:
Makes sense, thanks for explanation😀

Reply:
As for this, if you do want to keep some logging messages in for when Zellij itself is starting (I think that could be a nice way to see where sessions begin and end in the log file), maybe we could try to find a way to print separate messages for client and server startup?